spec/requests/teachers/registrations_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'swagger_helper'
+
+describe 'Teacher Registrations API', type: :request, swagger_doc: 'v1/swagger.json' do
+  path '/teachers' do
+    post 'Registers a new teacher' do
+      tags 'Teacher Registration'
```

Comment:
Curiousity: what does swagger do with `tags` ?

Reply:
In the documentations UI, it uses tags as the label for the API end points